cartography/intel/github/repos.py

```diff
@@ -49,11 +51,23 @@
                         login
                         __typename
                     }
+                    requirements:object(expression: "HEAD:requirements.txt") {
+                        ... on Blob {
+                            text
+                        }
+                    }
+                    requirements3:object(expression: "HEAD:requirements3.txt") {
```

Comment:
My plan is to keep requirements.txt parsing standard in open source, and since requirements3.txt is just a Lyft thing we'll roll our own function to do that internally; we already have patterns in place for this sort of customization. I do want to keep requirements.txt parsing in OSS even if not all users use Python because there is negligible perf difference if they don't have requirements.txts and it shows them an example on how they can parse blobs in GitHub repos.

Reply:
Sounds good.